MdsSupportingClasses/MdsFields.php

```diff
@@ -185,6 +185,13 @@ public static function getFields()
                 ),
                 'default' => 24,
             ],
+            'enable_town_suburb_search'=> [
+                'title' => __('Enable Town Suburb Search'),
+                'type' => 'checkbox',
+                'default' => 'yes',
```

Comment:
If the default is yes, but we only render the suburb/town field based on `$this->settings->getValue('enable_town_suburb_search') == 'yes';` 
Do we not need to populate the database with it on install?
Else the field will not appear by default after the plugin is installed (If they haven't saved any settings yet)

Reply:
No need to pre populate the db, I defaulted to false and the change will only  be effective once the user updates the configurations